shared/src/packages/orders/libs/types/order-find-by-request-dto.type.ts

```diff
@@ -0,0 +1,5 @@
+import { type OrderEntity } from './order-entity.type.js';
+
+type OrderFindByRequestDto = Partial<Omit<OrderEntity, 'id'>>;
```

Comment:
won't it be suitable?
```suggestion
type OrderFindByRequestDto = Omit<OrderEntity, 'id'>;
```

Reply:
I tried to make universal method that can update different fields, and some can be present, some not if at some point we know that we need only some field it can be more stricted

Reply:
fixed only to driverId, userId and businessId